tsinfer/formats.py

```diff
@@ -2007,30 +2064,69 @@ def num_alleles(self, sites=None):
     def variants(self, sites=None):
```

Comment:
Let's add a ``recode_ancestral=True`` option here. If False, we just return the alleles/genotypes exactly as they are.

Reply:
Yes, good call. I was going to suggest this as an update too.